avalanche/training/strategies/base_online_strategy.py

```diff
@@ -0,0 +1,173 @@
+from typing import Optional, Sequence, List, Union
+
+import copy
+import torch
+from torch.nn import Module, CrossEntropyLoss
+from torch.optim import Optimizer
+
+from avalanche.training.plugins.evaluation import default_logger
+from avalanche.training.plugins import StrategyPlugin, EvaluationPlugin
+from avalanche.training.strategies.base_strategy import BaseStrategy
+from avalanche.benchmarks.scenarios import Experience
+from avalanche.benchmarks.utils.avalanche_dataset import AvalancheSubset
+from avalanche.models import DynamicModule
+
+
+class BaseOnlineStrategy(BaseStrategy):
+    def __init__(self, model: Module, optimizer: Optimizer,
+                 criterion=CrossEntropyLoss(), num_passes: int = 1,
+                 train_mb_size: int = 1,
+                 eval_mb_size: int = None, device=None,
+                 plugins: Optional[List[StrategyPlugin]] = None,
+                 evaluator: EvaluationPlugin = default_logger, eval_every=-1):
+        super().__init__(
+            model, optimizer, criterion,
+            train_mb_size=train_mb_size, train_epochs=1,
+            eval_mb_size=eval_mb_size, device=device, plugins=plugins,
+            evaluator=evaluator, eval_every=eval_every)
+
```

Comment:
can you add a warning inside the constructor?
Something like: "This is an unstable experimental strategy. You can check https://github.com/ContinualAI/avalanche/blob/master/examples/cope.py for an alternative approach to online CL (data incremental scenario)."